examples/images/cifar10/compute_fid.py

```diff
@@ -51,7 +51,7 @@
 # Load the model
 PATH = f"{FLAGS.input_dir}/{FLAGS.model}/{FLAGS.model}_cifar10_weights_step_{FLAGS.step}.pt"
 print("path: ", PATH)
-checkpoint = torch.load(PATH)
+checkpoint = torch.load(PATH, map_location=device)
```

Comment:
would the code work if you run inference over 1 GPU while the network was trained on 2?

Reply:
Yep. I'd trained on 2 GPUs, but done inference on only 1 GPU. If I remember correctly, without `map_location=device`, an error was thrown when doing inference on only 1 GPU, while training was done on multiple GPUs.

Reply:
Awesome. Thank you for trying this.